plugins/civo/test-fixtures/.civo.json

```diff
@@ -0,0 +1,11 @@
+{
+    "apikeys": {
+      "newspidey": "XFIx85McyfCQc490j1tBa5b5s2XiWerNdOdfnkrOnchEXAMPLE"
+    },
+    "meta": {
+    
+      "current_apikey": "newspidey1",
+      "default_region": "LON1"
+   
+    }
+  }
```

Comment:
nit: let's add a new line at the ending of this file

Reply:
sure!